.github/workflows/label-failed-test.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  issue_commented:
+    name: Skip failed test on comment
```

Comment:
nit - name looks pulled from the skip workflow

Reply:
Thanks, fixed